src/po-theme-custom.css

```
@@ -868,12 +868,12 @@ po-link {
   MENU
 \*------------------------------------*/
 
-  --color-menu-color-menu-collapse-icon: var(--color-primary);
-  --color-menu-color-menu-item-text: var(--color-primary);
+  --color-menu-color-menu-collapse-icon: var(--color-brand-01-lightest);
+  --color-menu-color-menu-item-text: var(--color-brand-01-lightest);

```

verificar na outras variáveis do menu. importante para manter a customizaçÃo.
```suggestion
  --color-menu-color-menu-collapse-icon: var(--color-primary-light-95);
  --color-menu-color-menu-item-text: var(--color-primary-light-95);

```